src/components/NVD3LineChart.ts

```diff
@@ -49,8 +49,8 @@ class NVD3LineChart extends Component<Nvd3LineChartProps, {}> {
             style.height = `${this.props.height}%`;
         }
 
-        return DOM.div({ className: "widget-time-series nv-chart", style },
-            DOM.svg({ ref: node => this.svg = node })
+        return createElement("div", { className: "widget-time-series nv-chart", style },
+            createElement("svg", { ref: node => this.svg = node })
```

Comment:
when `svg` above is SVGElement, u can make the node be returned as SVGElement by doing this
`ref: node => this.svg = node as SVGElement`